wagtail/contrib/search_promotions/tests.py

```diff
@@ -1560,7 +1558,8 @@ def test_equivalent_queries(self):
     def test_different_queries(self):
         queries = [
             "HelloWorld",
-            "HelloWorld!" "  Hello  World!  ",
```

Comment:
I'm pretty sure this was missing a comma in the original commit 3754d34caaed1392c8c7696d6c76a7bb410cf04b (#5418)